<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<widget class="QWidget" name="stepsTab">
  <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
    </sizepolicy>
    </property>
    <property name="minimumSize">
    <size>
    <width>200</width>
    <height>278</height>
    </size>
    </property>
    <property name="autoFillBackground">
    <bool>false</bool>
    </property>
    <attribute name="title">
    <string>Steps</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="topMargin">
    <number>11</number>
    </property>
    <item>
    <widget class="QTreeWidget" name="stepsTreeWidget">
    <column>
        <property name="text">
        <string>#</string>
        </property>
    </column>
    <column>
        <property name="text">
        <string>Operation</string>
        </property>
    </column>
    <column>
        <property name="text">
        <string>Target</string>
        </property>
    </column>
    </widget>
    </item>
    <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
        <widget class="QPushButton" name="stepDownBtn">
        <property name="text">
        <string>&lt;</string>
        </property>
        </widget>
    </item>
    <item>
        <widget class="QPushButton" name="stepUpBtn">
        <property name="text">
        <string>&gt;</string>
        </property>
        </widget>
    </item>
    <item>
        <widget class="QPushButton" name="removeStepBtn">
        <property name="text">
        <string>-</string>
        </property>
        </widget>
    </item>
    <item>
        <widget class="QPushButton" name="addStepBtn">
        <property name="text">
        <string>+</string>
        </property>
        </widget>
    </item>
    <item>
        <widget class="QPushButton" name="runBtn">
        <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
        </sizepolicy>
        </property>
        <property name="minimumSize">
        <size>
        <width>0</width>
        <height>0</height>
        </size>
        </property>
        <property name="font">
        <font>
        <bold>false</bold>
        </font>
        </property>
        <property name="text">
        <string>Run</string>
        </property>
        <property name="checkable">
        <bool>false</bool>
        </property>
        <property name="flat">
        <bool>false</bool>
        </property>
        </widget>
        </item>
    </layout>
    </item>
    </layout>
  </widget>
</ui>